Wikipedia/Code/WMFDatabaseHouseKeeper.swift

```diff
@@ -109,23 +111,31 @@ import Foundation
         */
         
         let articlesToDeleteFetchRequest = WMFArticle.fetchRequest()
-        var articlesToDeletePredicate = NSPredicate(format: "viewedDate == NULL && savedDate == NULL && placesSortOrder == NULL && isExcludedFromFeed == %@", NSNumber(booleanLiteral: false))
+        var articlesToDeletePredicate = NSPredicate(format: "viewedDate == NULL && savedDate == NULL && placesSortOrder == 0 && isExcludedFromFeed == FALSE")
```

Comment:
You're sure that `FALSE` works? I did it the only way due to the way it was in ObjC and googling around, but that's obviously much simpler if it has the same effect